app/notebook/server/CalcWebSocketService.scala

```diff
@@ -144,7 +146,17 @@ class CalcWebSocketService(
       case Terminated(actor) =>
         Logger.debug("Termination of op calculator")
         if (actor == calculator) {
-          spawnCalculator()
+          Logger.error(s"Remote calculator ($calculator) has been terminated !!!!!")
```

Comment:
I'd vote for eventually renaming `calculator` into something like `executor process`. let's think of even better names :)